src/print-1-10-growing.py

```diff
@@ -1,2 +1,12 @@
 
 # Print the numbers described in the exercise
+list = " "
```

Comment:
You probably don't want to call a string `list`. That will confuse the heck out of anyone reading your code (including yourself).